src/check/parse/tokenize.zig

```diff
@@ -855,8 +875,8 @@ pub const Tokenizer = struct {
             switch (b) {
                 // Whitespace & control characters
                 0...32, '#' => {
-                    if (self.cursor.chompTrivia()) |indent| {
-                        self.output.pushNewline(indent);
+                    if (self.cursor.chompTrivia()) |_| {
+                        self.output.pushNewline(self.cursor.popComment());
```

Comment:
Just to clarify this, newlines instead of store the indent now store the index to any comments that appear between lines?

Reply:
They store the start and endposition